x-pack/plugins/enterprise_search/public/applications/app_search/components/setup_guide/setup_guide.tsx

```diff
@@ -83,92 +84,99 @@ export const SetupGuide: React.FC<> = () => {
-                            </EuiText>
-                          ),
-                        },
-                        {
-                          title: 'Reload your Kibana instance',
-                          children: (
-                            <EuiText>
-                              <p>Run this code snippet to install things.</p>
-                              <EuiCodeBlock language="bash">npm install</EuiCodeBlock>
-                            </EuiText>
-                          ),
-                        },
-                      ]}
-                    />
-                  </>
+                title: 'Add your App Search host URL to your Kibana configuration',
+                children: (
+                  <EuiText>
+                    <p>
+                      Within your <EuiCode>config/kibana.yml</EuiCode> file, add the following the
+                      host URL of your App Search instance as{' '}
+                      <EuiCode>enterpriseSearch.host</EuiCode>. For example:
+                    </p>
+                    <EuiCodeBlock language="yml">
+                      enterpriseSearch.host: &apos;http://localhost:3002&apos;
+                    </EuiCodeBlock>
+                  </EuiText>
                 ),
               },
               {
-                id: 'smas',
-                name: 'Self-Managed',
-                content: (
+                title: 'Reload your Kibana instance',
+                children: (
+                  <EuiText>
+                    <p>
+                      After updating your Kibana config file, restart Kibana to pick up your
+                      changes.
+                    </p>
+                    <p>
+                      If you’re using{' '}
+                      <EuiLink
+                        href="https://swiftype.com/documentation/app-search/self-managed/security#elasticsearch-native-realm"
+                        target="_blank"
+                      >
+                        Elasticsearch Native
+                      </EuiLink>{' '}
+                      auth within App Search - you’re all set! All users should be able to use App
+                      Search in Kibana automatically, inheriting the existing access and permissions
+                      they have within App Search.
+                    </p>
+                  </EuiText>
+                ),
+              },
+              {
+                title: 'Troubleshooting issues',
+                children: (
                   <>
+                    <EuiAccordion
+                      buttonContent="App Search is on a different Elasticsearch cluster than Kibana"
+                      id="standard-auth"
+                      paddingSize="s"
+                    >
+                      <EuiText>
+                        <p>
+                          The plugin currently relies on App Search and Kibana sharing the same
```

Comment:
Unless you have an `enterprise_search` user with a matching password on both 😲, which is how I'm making the E2E tests work, currently.

Reply:
No action needed here, just commenting for curiosity.

Reply:
Hahaha oh man I hadn't realized that. That cracks me up. It's definitely not a good production-level solution though (since you'd need multiple users with the same pw across the 2 clusters??), more like a dev workaround for getting up and running, right?

Reply:
Right, totally not a production-level solution.